### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by kvet**

### CODE REVIEW

File: tools/rollup-utils.js [hunk 1]
```diff
@@ -40,9 +40,13 @@ const knownGlobals = {
...
-      const modulePkg = JSON.parse(readFileSync(require.resolve(join(moduleId, 'package.json'))));
-      return modulePkg.globalName;
+      if (moduleId in knownGlobals) return knownGlobals[moduleId];
+
+      try {
+        const modulePkg = JSON.parse(readFileSync(require.resolve(join(moduleId, 'package.json'))));
+        return modulePkg.globalName;
+      } catch (e) {
+        return null;
+      }
```
Comment (SergeyAlexeev) [inline tools/rollup-utils.js]: Is this change related to the new rollup version?

Comment (kvet) [reply tools/rollup-utils.js]: Yes. Also, I am not sure that this is not a bug.
